config/settings.yml

```diff
@@ -58,4 +58,6 @@ aws:
   region: 'us-west-2'
   access_key_id: 'fake-access'
   secret_access_key: 'fake-secret'
-  base_s3_bucket: 'sul-speech-to-text-dev'
+  speech_to_text:
+    base_s3_bucket: 'sul-speech-to-text-dev' # default bucket for storing speech-to-text file
+    sqs_new_job_queue_url: 'https://sqs.us-west-2.amazonaws.com/queue_url' # queue to send new speech-to-text jobs
```

Comment:
what about `sqs_done_queue_url`, to mirror the naming in https://github.com/sul-dlss/speech-to-text/pull/9?

Reply:
heh, i'd actually meant to say `sqs_todo_queue_url`, but thanks for adding the one i mentioned and doing the rename!